HHS/Head-Start-TTADP
# [Prod] CSV will include missing goals, fix topics on unlocked and changed reports, responsive menu improvements, fix bug with adding recipients, update objectives expander UI, Topics frequency graph performance

## Description of change
- Fixes a bug where the CSV export wouldn't include goals for reports that also had no objectives
- The text on the objectives expander within the recipient record now reflects whether or not the objectives are expanded
- Fixes a bug where adding a recipient to a report that already had goals in progress would clear out some fields
- Fixes a bug where the wrong topic data would be shown on the approved AR table if that report had been unlocked and a topic changed
- CSS improvements around the behavior of the side nav and the header bar/logo
- Updates various dependencies to stay ahead of what cloud.gov supports and vulnerable vendor code flagged by the static code analyzer
- Performance fix for the topics frequency graph on the regional dashboard.

## How to test
- Filter to report 23470 on staging. Export the CSV for that report and confirm that it includes all the goals for that report
- View the recipient record and toggle the visibility of the objectives. Note that language switches between "view" and "hide" objectives.
- Create a new report, adding a recipient and a goal and objective. After adding the goal and objective, return to the activity summary and add another recipient. Return to the goal and objective and confirm all your data remains
- Create and approve a report. Unlock the report and change the topics you used on that objective, and re-approve. Confirm that the approved activity reports table shows the correct, updated topics.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1490
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1413
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1566
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1560
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1557

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
